bedrock/lightbeam/templates/lightbeam/lightbeam.html

```diff
@@ -80,12 +80,6 @@ <h2 class="page-slogan">{{_('How Lightbeam Works')}}</h2>
       {{_('Not all tracking is bad. Many services rely on user data to provide relevant content and enhance your online experience.')}} {{_('But tracking can happen without the user’s knowledge.')}} {{_('That’s not okay for some.')}} {{_('It should be you who decides when, how and if you want your browsing data to be shared. We recognize the importance of transparency and our mission is all about empowering users — both with tools and information.')}}  <a href="{{ url('lightbeam.about') }}"> {{_('About Lightbeam')}} »</a>
     </p>
   </div>
-  <div class="span6">
```

Comment:
Changing the "Tracking & Privacy" block above to `span8` would also help pad things out a little more visually.